### DISCUSSION TITLE
[帮助请求]浏览器内显示空白

### DISCUSSION THREAD

**Discussion by MRUIAW** [mcthesw/game-save-manager#114] [Q&A] [answered]
大佬您好!
我是一位小白开发者, 请问在终端运行 `pnpm run tauri:dev` 后, 浏览器内为空是为何?
![image](https://github.com/mcthesw/game-save-manager/assets/119018519/4d4cc6c1-1b48-4f7a-87a9-468a9a7ca8ca)

**Comment by mcthesw** [comment]
欢迎！
这种情况因为前端内使用了Tauri提供的API，不能够直接在浏览器中运行，正常来说应该会启动一个Tauri窗口，那里面能够正常运行

**Comment by mcthesw** [comment, marked as answer]
我看到你图中终端提示已经退出，如果是意外的崩溃，并没有启动Tauri窗口，可能是因为没有安装Webview运行环境,详见[Tauri官方文档](https://tauri.app/v1/references/webview-versions/)，Windows系统下可以安装[这个](https://developer.microsoft.com/en-us/microsoft-edge/webview2/?form=MA13LH)

**Comment by MRUIAW** [comment]
问题解决了, 感谢!

**Reply by MRUIAW** [reply]
原因貌似是因为后台已经开着编译好的启动器导致的, 按照tarui文档所说, win11已经自带webview2环境了
